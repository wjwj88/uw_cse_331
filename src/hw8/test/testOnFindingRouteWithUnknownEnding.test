# this is to test the program recognize Unknown building
# whenever the ending building is unknown.
r
CSE
ABC

# terminate the program
q