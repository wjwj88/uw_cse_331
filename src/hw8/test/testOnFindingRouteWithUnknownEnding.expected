Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # this is to test the program recognize Unknown building 
# whenever the ending building is unknown. 
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: ABC

Enter an option ('m' to see the menu): 
# terminate the program